README.md

```diff
@@ -91,6 +91,11 @@ if you are interested to contribute to this repo.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
+## Weekly Bug Traige call
+
+We are conducting weekly bug triage call at our slack on every tuesday.
```

Comment:
s/tuesday/Tuesday/

"We conduct weekly bug triage calls at our slack channel on Tuesdays" ?

Reply:
@ajarr Done.. thanks !